src/client/cypress/e2e/editor/backfill.cy.js

```diff
@@ -21,14 +21,29 @@ describe("Backfill crud tests", () => {
 
     // start editing session
     startBoreholeEditing();
+  });
+
+  it("add, edit and delete backfills", () => {
```

Comment:
PP: Testtitel so formulieren, dass es mit dem "it" einen Satz bildet.  Haben wir sonst glaube ich meistens (?)
"It adds, edits and deletes..."